src/Ocelot/DownstreamRouteFinder/UrlMatcher/UrlPathPlaceholderNameAndValueFinder.cs

```diff
@@ -1,161 +1,132 @@
 using Ocelot.Responses;
 
-namespace Ocelot.DownstreamRouteFinder.UrlMatcher
+namespace Ocelot.DownstreamRouteFinder.UrlMatcher;
+
+public class UrlPathPlaceholderNameAndValueFinder : IPlaceholderNameAndValueFinder
 {
-    public class UrlPathPlaceholderNameAndValueFinder : IPlaceholderNameAndValueFinder
+    private const string PlaceHolderLeft = "{";
+    private const string PlaceHolderRight = "}";
```

Comment:
Can be simply `char`
```suggestion
    private const char PlaceHolderLeft = '{';
    private const char PlaceHolderRight = '}';
```
For readability better to rename to 
- `PlaceHolderLeft` &rarr; `LeftCurlyBracket`
- `PlaceHolderRight` &rarr; `RightCurlyBracket`